### DISCUSSION TITLE
Replacing a color using the command line

### DISCUSSION THREAD

**Discussion by beig** [libvips/libvips#3444] [General]
I want to replace a color from pngs (or tiffs)

Using:
`vips-8.12.2-Tue Jan 25 09:34:32 UTC 2022`

So far I tried the following
```
vips bandjoin test.png test.png test.png out.v //you cant use pipe on the cli if i read that right
vips ifthenelse "(r==0)&&(g==0)&&(b==0)" ... // struggling with the command
```

I couldn't find any examples using the cli - or should I just switch to some bindings?

**Comment by jcupitt** [comment]
Hi @beig,

The libvips command-line is deliberately very basic. For complex operations, it's best to use a language binding.
